tessdata.yaml

```diff
@@ -0,0 +1,112 @@
+package:
+  name: tessdata
+  version: 4.1.0
+  epoch: 0
+  description: Tesseract training data
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - curl-dev
+      - git
+
+data:
+  - name: langs
+    items:
+      afr:
+      ara:
+      aze:
+      bel:
+      ben:
+      bul:
+      cat:
+      ces:
+      chi_sim:
+      chi_tra:
+      chr:
+      dan:
+      deu:
+      eng:
+      enm:
+      epo:
+      equ:
+      est:
+      eus:
+      fin:
+      fra:
+      frk:
+      frm:
+      glg:
+      grc:
+      heb:
+      hin:
+      hrv:
+      hun:
+      ind:
+      isl:
+      ita:
+      ita_old:
+      jpn:
+      kan:
+      kat:
+      khm:
+      kor:
+      lav:
+      lit:
+      mal:
+      mkd:
+      mlt:
+      msa:
+      nld:
+      nor:
+      osd:
+      pol:
+      por:
+      ron:
+      rus:
+      slk:
+      slv:
+      spa:
+      spa_old:
+      sqi:
+      srp:
+      swa:
+      swe:
+      tam:
+      tel:
+      tgl:
+      tha:
+      tur:
+      ukr:
+      vie:
+
+vars:
+  package_name: tessdata
+  package_version: 4.1.0
```

Comment:
do we need this here?

Reply:
No I forgot to take that back out, ended up using the package version not vars.package version. thanks much for the catch